### DISCUSSION TITLE
Change input select?

### DISCUSSION THREAD

**Discussion by simsong** [MonitorControl/MonitorControl#1578] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

I have a monitor that has multiple inputs. I would like to use MonitorControl to change the input of the secondary monitor. This should be possible with DDC. Code to make this change is in the open source tool `ddcctl` but it does not seem to work on my OS (Senoma 14.5)

### Describe the solution you'd like

I would like to be able to change the input of the secondary monitor with a hotkey or with the menu control (and ideally from the command line, by sending a message to the app).

### Describe alternatives you've considered

I tried `ddcctl` but the version I had was reading the file `/Library/Preferences/com.apple.windowserver.plist` instead of `/Library/Preferences/com.apple.windowserver.displays.plist`. When I added a symlink, `ddcctl` crashed with:
```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[NSTaggedPointerString countByEnumeratingWithState:objects:count:]: unrecognized selector sent to instance 0x90e7add34cacd282'
```

**Comment by waydabber** [comment, marked as answer]
Hi there - this feature was added to [BetterDisplay](https://github.com/waydabber/BetterDisplay). Not planned in MonitorControl.

**Reply by simsong** [reply]
OMG, this is amazing and perfect. Thank you so much for the pointer!
